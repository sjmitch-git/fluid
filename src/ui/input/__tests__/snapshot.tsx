import { render } from '@testing-library/react'
import { Input } from '..'

it('renders Input unchanged', () => {
	const { container } = render(<Input type='text' />)

	expect(container).toMatchSnapshot()
})
