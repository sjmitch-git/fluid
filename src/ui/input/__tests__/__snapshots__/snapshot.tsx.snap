// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Input unchanged 1`] = `
<div>
  <input
    class="input form-input w-full rounded-md group border-gray-400 shadow-sm font-normal text-inherit leading-inherit required:bg-accent invalid:ring-danger disabled:bg-neutral"
    data-testid="input"
    type="text"
    value=""
  />
</div>
`;
